TimeFormat.java

```diff
@@ -12,6 +12,16 @@ public static void main(String[] args) {
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+        String amOrPm = "AM";

```

Comment:
bad indentation

Reply:
It doesn't look this way on my Vscode, for some reason it got messed up on Github. 
I just noticed this issue. Do you have an idea why it might happen?

Reply:
I don't, think it there is an issue in github. Some programming langs depend on indentation for functionality. 
Try and double check, in the future :)